server/src/main/java/org/eclipse/lsp/cobol/core/engine/CobolLanguageEngine.java

```diff
@@ -211,4 +228,24 @@ private SyntaxError constructErrorMessage(SyntaxError syntaxError) {
     }
     return syntaxError;
   }
+
+  private Map<String, Collection<Location>> collectVariableDefinitions(
+      Collection<Variable> definedVariables) {
+    Multimap<String, Location> definitions = HashMultimap.create();
+    definedVariables.stream()
+        .filter(it -> !FILLER_NAME.equals(it.getName()))
+        .filter(it -> Objects.nonNull(it.getDefinition()))
```

Comment:
Isn't it always true?

Reply:
Yes, you right. But probably, we added this to avoid Coverity issues :(
Anyway, this part must be deleted after the migration is done.